packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2507,6 +2507,21 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
 
   Element _parent;
 
+  // Custom implementation of `operator ==` optimized for the ".of" pattern
+  // used with `InheritedWidgets`.
+  @override
+  bool operator ==(Object other) => identical(this, other);
```

Comment:
is that not what the inherited operator == does?

Reply:
I believe so, but the linter says if you overload `hashCode` you must also overload `operator==`.

Reply:
fair enough. i guess this is no worse than an `// ignore` would be.